ct.sh

```diff
@@ -20,6 +20,7 @@ export CC_x86_64_fortanix_unknown_sgx=clang-11
 export CC_aarch64_unknown_linux_musl=/tmp/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc
 export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER=/tmp/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc
 export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_RUNNER=qemu-aarch64
+export RUSTFLAGS="-A ambiguous_glob_reexports"
```

Comment:
Please file a new GH ticket to change this in a few months and a comment pointing to it

Reply:
ticket [created](https://github.com/orgs/fortanix/projects/1/views/1?filterQuery=goo&reload=true&pane=issue&itemId=29955735)